docs/pages/operations-guide.md

```diff
@@ -371,361 +383,285 @@ Given the above configuration, let's assume `10.0.10.8` is being decommissioned
+Make any configuration changes to this `options.json` file. 
 
-Navigate to the agent you want to view either directly from a task by clicking the "Agent" item in the breadcrumb when viewing a task (this will go directly to the agent hosting the task), or by navigating through the "Agents" menu item at the top of the screen (you will need to select the desired agent from the list).
+If you installed this service with a prior version of DC/OS, this configuration will not have been persisted by the the DC/OS package manager. You can instead use the `options.json` file that was used when [installing the service](#initial-service-configuration). 
 
-In the Agent view, you'll see a list of frameworks with a presence on that Agent. In the left pane you'll see a plain link named "LOG". Click that link to view the agent logs.
+<strong>Note:</strong> You need to specify all configuration values in the `options.json` file when performing a configuration update. Any unspecified values will be reverted to the default values specified by the DC/OS service. See the "Recreating `options.json`" section below for information on recovering these values.
 
-[<img src="img/ops-guide-agent.png" alt="view of tasks running on a given agent" width="400"/>](img/ops-guide-agent.png)
+###### Recreating `options.json` (optional)
 
-### Logs via the CLI
+If the `options.json` from when the service was last installed or updated is not available, you will need to manually recreate it using the following steps.
 
-You can also access logs via the [DC/OS CLI](https://dcos.io/docs/latest/usage/cli/install/) using the `dcos task log` command. For example, lets assume the following list of tasks in a cluster:
+First, we'll fetch the default application's environment, current application's environment and the actual template that maps config values to the environment:
 
+1. Ensure you have [jq](https://stedolan.github.io/jq/) installed.
+1. Set the service name that you're using, in this example we'll use `dse`:
 ```bash
-$ dcos task
-NAME                  HOST        USER  STATE  ID
-broker-0              10.0.0.242  root    R    broker-0__81f56cc1-7b3d-4003-8c21-a9cd45ea6a21
-broker-0              10.0.3.27   root    R    broker-0__75bcf7fd-7831-4f70-9cb8-9cb6693f4237
-broker-1              10.0.0.242  root    R    broker-1__6bf127ab-5edc-4888-9dd3-f00be92b291c
-broker-1              10.0.1.188  root    R    broker-1__d799afdb-78bf-44e9-bb63-d16cfc797d00
-broker-2              10.0.1.151  root    R    broker-2__4a293161-b89f-429e-a22e-57a1846eb271
-broker-2              10.0.3.60   root    R    broker-2__76f45cb0-4db9-41ad-835c-2cedf3d7f725
-[...]
+$ SERVICE_NAME=dse
+```
+1. Get the version of the package that is currently installed (replacing `dse` with the name of your service below):
+```bash
```

Comment:
if you've set the environment variable above, I don't think you need the parenthetical remark above?